### DISCUSSION TITLE
How to import Bootstrap as NPM module if aliased?

### DISCUSSION THREAD

**Discussion by martinbean** [twbs/bootstrap#31460] [General]
I’m slowly migrating a project that uses Bootstrap 4 to Bootstrap 5, so have both versions installed side by side. I did that like this:

```json
{
  "devDependencies": {
    "bootstrap-4": "npm:bootstrap@^4.5.1",
    "bootstrap-5": "npm:bootstrap@5.0.0-alpha1"
  }
}
```

In my JavaScript file where I want to import Bootstrap 5, I’m doing this:

```js
import bootstrap from "bootstrap-5";
```

However, this throws the following error when trying to compile using Laravel Mix (which is basically a wrapper around Webpack):

```
export 'bootstrap' was not found in 'bootstrap-5'
```

How do I go about importing Bootstrap’s JavaScript if using an NPM alias like this?

**Comment by Johann-S** [comment]
you should import Bootstrap like that if you want to import everything:
```js
import 'bootstrap';
```

if you want to import separate components you can do:
```js
import { Modal } from 'bootstrap';
```
